src/lib/addKeyDownHandler.ts

```diff
@@ -0,0 +1,40 @@
+import * as _ from 'lodash'
+import keyboardHandlerFilter from './accessibility/Helpers/keyboardHandlerFilter'
+
+/**
+ * Adds onKeyDown handler to the Component's rest props, based on Component's actions
+ * and keys mappings defined in Accessibility behavior
+ * @param {Object} rest The rest props which is to be extended by adding onKeyDown handler
+ * @param {AccessibilityActions} actions The input element which is to loose focus.
+ * @param {IAccessibilityDefinition} accessibility The input element which is to loose focus.
+ * @param {IRenderConfigProps} props The props which are used to invoke onKeyDown handler passed from top.
+ */
+const addKeyDownHandler = (rest, actions, accessibility, props) => {
+  const actionsDefinition = accessibility.actionsDefinition
+
+  if (!actions || !actionsDefinition) return
+
+  let hasCommonActions = false
+  for (const actionName in actionsDefinition) {
```

Comment:
Not significantly shorter, but easier to read?
```js
const hasCommonActions = _.intersection(_.keys(actionsDefinition), _.keys(actions)).length > 0
```

Reply:
Definitely, thanks! Still can't get used to lodash, like vanilla :)